site2/docs/concepts-messaging.md

```diff
@@ -203,6 +203,12 @@ Client libraries provide listener implementation for consumers. For example, the
 
 ### Acknowledgment
 
+:::note
+
+By default, the acknowledge timeout is disabled and that means that messages delivered to a consumer will not be re-delivered unless the consumer crashes.
+
+:::
```

Comment:
It's better to move to the `Acknowledgment timeout` section

https://github.com/apache/pulsar/pull/18436/files#diff-2c2475e441b7e22ba41b8e66cc773464a4d54deeba8380862316971ce4f0b95cR307

Reply:
You are right. Changed.